### DISCUSSION TITLE
Dummy Alarm

### DISCUSSION THREAD

**Discussion by thechrisreese** [dgreif/ring#581] [Ideas]
Would it be possible to create a dummy ring alarm like homebridge-securitysystem creates a dummy alarm?

It would be great to be able to pass home and away to my cameras without a real alarm system in place and have the siren setting as well to pass through to audio files.

**Comment by dgreif** [comment]
This is currently possible if you have location modes enabled in the Ring app. If you do, you should see an alarm like control that lets you set home and away, matching behavior for location modes. Do your have modes enabled? If so, can you confirm you see the alarm control in HomeKit?

**Comment by thechrisreese** [comment]
Ok I have the mode working 

![image](https://user-images.githubusercontent.com/20584219/109770820-e4eb0480-7bb0-11eb-877a-0de196951129.jpeg)

But is there a way to spoof the siren like the homebridge-security system 

![image](https://user-images.githubusercontent.com/20584219/109770881-ff24e280-7bb0-11eb-8969-49613eb6cb3e.jpeg)

It helps trigger alarms or connect to the dummy sensors to send notifications for Alexa.

**Reply by dgreif** [reply]
Sorry, I'm not quite sure what you are asking for. The camera sirens can all be controlled individually and are exposed as accessories within each camera. You can create a scene to turn them all on at the same time if that's your goal. Is there something else you are trying to accomplish that?
